app/src/main/kotlin/com/wire/android/ui/authentication/create/CreateAccountFlowType.kt

```diff
@@ -0,0 +1,30 @@
+package com.wire.android.ui.authentication.create
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import com.wire.android.R
+
+enum class CreateAccountFlowType(
+    val routeArg: String = "",
+    @StringRes val titleResId: Int,
+    @StringRes val summaryTextResId: Int = 0,
+    @DrawableRes val summaryIconResId: Int = 0,
+    ) {
+    CreatePersonalAccount(
+        routeArg =  "create_personal_account",
+        titleResId = R.string.create_personal_account_title,
+        summaryTextResId = R.string.create_personal_account_summary_text,
+        summaryIconResId = R.drawable.ic_create_personal_account_success,
+    ),
+    CreateTeam(
+        routeArg = "create_team",
+        titleResId = R.string.create_personal_account_title, //TODO change
+        summaryTextResId = R.string.create_personal_account_summary_text, //TODO change
+        summaryIconResId = R.drawable.ic_create_personal_account_success, //TODO change
+        ),
+    None(titleResId = R.string.set_username_title);
```

Comment:
I don't understand why None is used here for

Reply:
I separated both situations and changed it to `AppStart` - the situation where app detects at the start that this user haven't set his handle yet 🚀